bots/tep-automation/pkg/tep/types.go

```diff
@@ -0,0 +1,183 @@
+package tep
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+	"time"
+)
+
+const (
+	// ActionParamName is the param name that will hold the pull request webhook action
+	ActionParamName = "pullRequestAction"
+	// PRNumberParamName is the param name that will hold the pull request number
+	PRNumberParamName = "pullRequestNumber"
+	// PRTitleParamName is the param name that will hold the pull request title
+	PRTitleParamName = "pullRequestTitle"
+	// PRBodyParamName is the param name that will hold the pull request body
+	PRBodyParamName = "pullRequestBody"
+	// PackageParamName is the param name that will hold the pull request's repository owner/name
+	PackageParamName = "package"
+	// PRIsMergedParamName is the param name that will hold whether the pull request is merged
+	PRIsMergedParamName = "pullRequestIsMerged"
+	// GitRevisionParamName is the param name that will hold the HEAD git SHA for the PR
+	GitRevisionParamName = "gitRevision"
+
+	// CommunityBlobBaseURL is the base URL for links to blobs in github.com/tektoncd/community/teps
+	CommunityBlobBaseURL = "https://github.com/tektoncd/community/blob/main/teps/"
+
+	// DefaultTrackingIssueDescription is used in the tracking issue when no description already exists in the issue body.
+	DefaultTrackingIssueDescription = "(specify a one-line description of the TEP)"
+	// DefaultTrackingIssueField is used in the tracking issue when no release target (for either alpha or beta)
+	// or project(s) already exists in the issue body.
+	DefaultTrackingIssueField = "(unspecified)"
+
+	// TrackingIssueBodyTmpl is the template (using text/template) for the body of tracking issues.
+	TrackingIssueBodyTmpl = `This issue tracks TEP-{{ .issue.TEPID }}.
```

Comment:
This is the template for the issue body, based on @jerop's manually-created issue at https://github.com/tektoncd/community/issues/601. I didn't break down the `TEP References` into `alpha` and `beta` here, because I couldn't see a way to detect when a TEP or implementation PR is for `alpha` or for `beta`. The `Description`, `Projects`, and `Release Targets` "fields" are set manually via editing the issue in GitHub, not automatically from TEP markdown or PR information, but the values are preserved when the bot makes other changes to the issue body.